src/modules/thirdParty/sagas.ts

```diff
@@ -237,9 +246,24 @@ export function* thirdPartySaga(builder: BuilderAPI, catalyst: CatalystClient) {
         throw new Error('Invalid Identity')
       }
 
+      yield put(
```

Comment:
I think we might want to move this put to the sagas that controls the flow of the deployment as most of the modal changes are there, is it possible to do this?

Reply:
That was my first intention but the sagas that controls the flow doesn't know when the user starts the deploy, it only knows when it's done. I tried to add one step in the middle but controlling the flow from here was more straightforward. It could be triggered from the UI component as well, but I couldn't find a way to do it from the sagas that controls the flow.